src/monattack.cpp

```diff
@@ -2785,14 +2786,32 @@ bool mattack::photograph( monster *z )
         }
     }
 
-    if( z->friendly ) {
-        // Friendly (hacked?) bot ignore the player.
+    if( z->friendly || g->u.weapon.typeId() == "e_handcuffs" ) {
+        // Friendly (hacked?) bot ignore the player. Arrested suspect ignored too.
         // TODO: might need to be revisited when it can target npcs.
         return false;
     }
     z->moves -= 150;
     add_msg( m_warning, _( "The %s takes your picture!" ), z->name().c_str() );
     // TODO: Make the player known to the faction
+    if( one_in( 6 ) ) {
+        sounds::sound( z->pos(), 15, sounds::sound_t::alert,
+                       _( "a robotic voice boom, \"Citizen " ) + Name::generate( g->u.male ) + "!\"" );
```

Comment:
a robotic voice *booms*

Reply:
Are you sure? Player side would be "You hear a robotic voice booms, XXX" instead of "You hear a robotic voice boom, XXX"

Reply:
Yeah, never mind. I should familiarize myself more before I open my mouth. :)